### PR TITLE
fix(pinpoint): add campaign attributes to push events

### PR DESCRIPTION
**Pull request by raphkim**
*Description of changes:*
Journey implementation assumed that journey and campaign events were mutually exclusive and incorrectly removed campaign attributes even when they were expected in the payload. This PR reverses that change.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by changxu0306 — APPROVED** [review]

**Comment by richardmcclellan** [comment]
Is this fixing a regression introduced by https://github.com/aws-amplify/aws-sdk-android/pull/1542/?    I was looking at the changes in that PR, and it doesn't seem like this is reverting anything?  Kinda hard to follow the previous PR though.  Could you just explain how this fix works?

**Comment by raphkim** [comment]
#1542 renamed `setCampaignAttributes()` method to `updateEventSourceGlobally()` and put that method inside the `if-else` statement that checks whether the app is in foreground or not (thereby adding the attributes only if the app is in the background). This PR calls the new `updateEventSourceGlobally()` unconditionally again.

**Review by richardmcclellan — APPROVED** [review]
